libs/androidlib/test/src/mill/androidlib/KeytoolTest.scala

```diff
@@ -0,0 +1,87 @@
+package mill.androidlib
```

Comment:
can move this to keytool package to match the same package as the Keystore being tested, so it also won't need the 
`import mill.androidlib.keytool.*`